src/Core/src/Platform/Android/ImageButtonExtensions.cs

```diff
@@ -14,7 +17,32 @@ internal static void UpdateBackground(this ShapeableImageView platformButton, II
 		{
 			Paint? paint = imageButton.Background;
 
-			platformButton.Background = paint?.ToDrawable(platformButton.Context);
+			if (paint is not null)
+			{
+				// TODO: Assign Ids to detect if we already created the RippleDrawable to just
```

Comment:
@PureWeen @mattleibow Different Views are used for Button and ImageButton and they have classes with different extension methods as well. However, the fix may be similar to Button one. Wrap the drawable with the Background with a RippleDrawable that we only create once, and subsequent updates, we modify the one created.

Thoughts?